tests/test-data/dir-nested.block

```diff
@@ -1,5 +1,4 @@
 5
-" $��G�,�A�4{���x�Z/.����D`�200Bytes.txt�3
```

Comment:
@noffle could you redo this PR with just the README improvs. @nginnever PR will be merged soon which adds the expected behaviour on the test. Sorry that you had to go through that trouble.